src/multi.rs

```diff
@@ -192,6 +215,38 @@ impl MultiState {
         drawable.draw()
     }
 
+    pub(crate) fn println<I: AsRef<str>>(&mut self, msg: I) -> io::Result<()> {
+        let msg = msg.as_ref();
+
+        // If msg is "", make sure a line is still printed
+        let lines: Vec<String> = match msg.is_empty() {
+            false => msg.lines().map(Into::into).collect(),
+            true => vec![String::new()],
+        };
+
+        if self.draw_target.is_hidden() {
```

Comment:
This isn't needed anymore (`draw_target.drawable()` will return `None` in this case, anyway).